index.html

```diff
@@ -11,6 +11,12 @@
 			content="A smart shopping list that learns your purchase habits and makes suggestions, so you don't forget to buy what's important."
 		/>
 		<link rel="icon" type="image/svg+xml" href="/src/favicon.ico" />
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+			integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN"
+			crossorigin="anonymous"
+		/>
```

Comment:
This works great! Thanks for integrating it in! I can see the default styles (that we can alter later on ofc), and didn't notice anything breaking. 🚀

Reply:
Yeah, I wanted tojust get it and and using it to make sure that 
1. it worked and didn't break everything 😂😅
2. we would still have options for css (like the sass suggestion you made!)